omod/src/main/webapp/resources/htmlforms/096a-HealthUnitTBRegister-SummaryPage.xml

```diff
@@ -663,7 +841,6 @@
 					  			id="transfer-in-unit-tb-no" required="false" />

```

Comment:
@moshonk I have found that the Transfer-In-UnitTB number is an identifier and not an obs ...